src/topdup_open/topdup_app/routes.py

```diff
@@ -36,6 +38,15 @@ def home():
             p.domain = re.search(_config.split, p.url).group(1)
     return render_template('index.html', posts=posts, all_posts=all_posts)
 
+@app.route("/compare_doc")
```

Comment:
về convention cho url, em nghĩ nên hạn chế sử dụng underscore, có thể sử dụng dashes trong trường hợp thực sự cần thiết. Anh có thể tham khảo [ở đây](https://developers.google.com/search/docs/advanced/guidelines/url-structure?hl=en&visit_id=637453228355683042-3342644942&rd=1).
Trong trường hợp này em nghĩ mình nên để object lên trước và sau đó là action với object đó 
```suggestion
@app.route("/docs/compare")
```